examples/ts-react-ui-chatbot/src/styles.css

```
@@ -0,0 +1,98 @@
+  font-style: normal;
+  font-weight: 300;
+  font-display: swap;
+  src: url('/brand/fonts/Inter-latin.woff2') format('woff2');
+}
+
+@custom-variant dark (&:is(.dark *));
+
+@theme inline {
+  --color-background: var(--background);
+  --color-foreground: var(--foreground);
+  --font-sans: Inter, ui-sans-serif, system-ui, sans-serif;
```

Stylelint (17.14.0)

[error] Expected "Inter" to be "inter" (value-keyword-case)

(value-keyword-case)